src/utils/webview.ts

```diff
@@ -0,0 +1,34 @@
+/*
+* This program and the accompanying materials are made available under the terms of the *
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at *
+* https://www.eclipse.org/legal/epl-v20.html                                      *
+*                                                                                 *
+* SPDX-License-Identifier: EPL-2.0                                                *
+*                                                                                 *
+* Copyright Contributors to the Zowe Project.                                     *
+*                                                                                 *
+*/
+
+import * as fs from "fs";
+import * as path from "path";
+
+export function getContentForWebView(directory: string): {html: string; js: string; css: string;} {
+    return {
+        html: fs.readFileSync(path.resolve(directory, "index.html"), "UTF-8"),
+        js: fs.readFileSync(path.resolve(directory, "index.js"), "UTF-8"),
+        css: fs.readFileSync(path.resolve(directory, "index.css"), "UTF-8")
```

Comment:
Can we `await fs.promises.readFile`? Doing a sycnchronous read blocks the event loop for everyone. https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_fs_promises_api

Reply:
Fixed.